pallets/schemas/src/migration/v4.rs

```diff
@@ -14,21 +14,23 @@ use sp_runtime::TryRuntimeError;
 use sp_std::{collections::btree_map::BTreeMap, vec::Vec};
 
 /// get known schema names for mainnet
-pub fn get_known_schemas<T: Config>() -> BTreeMap<SchemaId, Vec<u8>> {
+pub fn get_known_schemas<T: Config>() -> BTreeMap<SchemaId, Vec<u8>, boolean> {
 	let genesis_block: BlockNumberFor<T> = 0u32.into();
 	let genesis = <frame_system::Pallet<T>>::block_hash(genesis_block);
 
 	if let DetectedChainType::FrequencyMainNet =
 		get_chain_type_by_genesis_hash(&genesis.encode()[..])
 	{
 		// only return what needs to change which for this case is only on mainnet
+		// (schema_id, name, clear prior versions)
 		BTreeMap::from([
-			(5, b"dsnp.update".to_vec()),
-			(6, b"dsnp.profile".to_vec()),
-			(7, b"dsnp.public-key-key-agreement".to_vec()),
-			(8, b"dsnp.public-follows".to_vec()),
-			(9, b"dsnp.private-follows".to_vec()),
-			(10, b"dsnp.private-connections".to_vec()),
+			(5, b"dsnp.update".to_vec(), true),
+			(19, b"dsnp.update".to_vec(), false),
```

Comment:
This is the one that broke the migration. So to fix that, we clear it out the line before (replacing the bad one) and the re-add 19 to the end of the list